avocado/core/sysinfo.py

```diff
@@ -202,21 +206,28 @@ def run(self, logdir):
         logf_path = os.path.join(logdir, self.logf)
         stdin = open(os.devnull, "r")
         stdout = open(logf_path, "w")
-        self.pipe = subprocess.Popen(shlex.split(self.cmd), stdin=stdin, stdout=stdout,
-                                     stderr=subprocess.STDOUT, shell=False, env=env)
+        try:
+            self.daemon_process = subprocess.Popen(shlex.split(self.cmd),
+                                                   stdin=stdin, stdout=stdout,
+                                                   stderr=subprocess.STDOUT,
+                                                   shell=False, env=env)
+        except OSError:
+            log.debug("Not logging  %s (command not found) " % self.cmd)
```

Comment:
Since the exception being caught here is not explicitly the `FileNotFoundError`, what do you think to update the log message replacing `(command not found)` by `(command could not be run)`?

Reply:
@cacarrara yes make sense will adopt , thanks for review